docs/docs/50-user-guide/60-reference-docs/70-annotations.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents certain annotations and labels that are applicable to
+Kargo resource types and other Kubernetes resource types (e.g. `Secret`s or
+`ServiceAccounts`) and affect how Kargo handles those resources. The subset
+documented here are those that the maintainers have judged most likely to be of
+use to advanced users who may, for instance, be looking to interact
+programmatically with Kargo.
+
+## Annotations
+
+| Key | Target Resource(s) | Possible Values | Description |
+|-----|--------------------|-----------------|-------------|
+| `kargo.akuity.io/refresh` | `Warehouse`, `Stage`, `Promotion` | A string that is unique or at least unlikely to repeat, such as a UUID or a timestamp of "now" | Triggers reconciliation of the resource when its value changes. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the current `Freight`. |
+| `kargo.akuity.io/abort` | `Stage` | Verification ID | Aborts an in-progress `Freight` verification. |
+| `kargo.akuity.io/description` | Any | Any string | Human-readable description displayed in the UI. |
+| `kargo.akuity.io/color` | `Stage` | Hex color code (e.g. `#ff8800`) | Optional cosmetic color used in the UI's pipeline view. |
+| `kargo.akuity.io/authorized-stage` | `Argo CD Application` | `<project>:<stage>` | Indicates which `Stage` is authorized to manage the `Application` resource. |
+| `rbac.kargo.akuity.io/managed` | `ServiceAccount`, `Role`, `RoleBinding` | `"true"` |  Permits the UI or CLI (via the API server) to programmatically manage trios of `ServiceAccount`, `Role`, and `RoleBinding` resources via Kargo's own ["roles" abstraction](../50-security/20-access-controls/index.md#managing-mappings-and-permissions). Omit this annotation if you wish to exclusively manage these resources [declaratively](../50-security/20-access-controls/index.md#managing-kargo-roles-declaratively). |
+| `rbac.kargo.akuity.io/claim.<name>` | `ServiceAccount` | Claim name as suffix | Maps an OIDC claim (e.g., `sub`, `email`, or `groups`) to a `ServiceAccount`, enabling user-to-ServiceAccount mappings. For more details, refer to the access control sections of the [Operator Guide](../../40-operator-guide/40-security/30-access-controls.md) and [User Guide](../50-security/20-access-controls/index.md). |
```

Comment:
"Claim name as suffix" does not correctly describe possible values. It actually describes the _key_. Perhaps this should be "A claim value"